environment/commands/stop/all.js

```diff
@@ -1,32 +1,52 @@
-const ProcessManager = require('pm2-promise');
+const ProcessManager = require('pm2-promise')
 
-const { StopLocalNearNodeCommand } = require('./near.js');
+const { StopLocalNearNodeCommand } = require('./near.js')
 
 async function stop(serviceName) {
-    try {
-        await ProcessManager.delete(serviceName);
-        console.log(serviceName, 'successfully stopped...');
-    } catch (err) {
-        if (!err.message.includes('process or namespace not found')) {
-            console.log(serviceName, 'error stopping the process due to:', err);
-        } else {
-            console.log(serviceName, 'already stopped');
-        }
-    };
+  try {
+    console.log('Stopping', serviceName)
+    await ProcessManager.delete(serviceName)
+    console.log(serviceName, 'successfully stopped')
+  } catch (err) {
+    if (
+      err.message &&
+      err.message.indexOf('process or namespace not found') >= 0
+    ) {
+      console.log(serviceName, 'already stopped')
+    } else {
+      console.log(
+        serviceName,
+        'error stopping the process due to:',
+        err.message
+      )
+    }
+  }
 }
 
 class StopAllCommands {
-    static async execute() {
-        console.log('Stopping all processes...');
-
-        StopLocalNearNodeCommand.execute();
-        for (const serviceName of ['ganache', 'eth-relay', 'near-relay', 'near-watchdog']) {
-            await stop(serviceName);
-        }
+  static async execute() {
+    const consoleError = console.error
+    // A cool hack to avoid annoying pm2 printing to stderr
```

Comment:
Cool i didn't know :) btw we use forked version of https://github.com/near/pm2-promise only to remove this, with your change it can be changed back: https://github.com/near/rainbow-bridge/blob/master/environment/package.json#L32

Reply:
I returned back pm2-promise.